osu.Game/Users/UserListPanel.cs

```diff
@@ -28,7 +28,7 @@ public UserListPanel(APIUser user)
         [BackgroundDependencyLoader]
         private void load()
         {
-            Background.Width = 0.5f;
+            Background!.Width = 0.5f;
```

Comment:
Yeah no please don't do this. `Debug.Assert()` is much preferred.

That said if you go with my other `CreateBackground()` suggestion then this all can be refactored out to not require any of these gymnastics. It *will* result in some code duplication but I think it's fine.